core/src/test/java/org/apache/iceberg/inmemory/TestInMemoryInputFile.java

```diff
@@ -22,16 +22,17 @@
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import org.assertj.core.api.Assertions;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class TestInMemoryInputFile {
   @Test
   public void testReadAfterClose() throws IOException {
     InMemoryInputFile inputFile =
         new InMemoryInputFile("abc".getBytes(StandardCharsets.ISO_8859_1));
     InputStream inputStream = inputFile.newStream();
-    Assert.assertEquals('a', inputStream.read());
+    Assertions.assertThat(inputStream.read())
+        .as("Checking read value from inputStream")
```

Comment:
do we really need this description?

Reply:
I think it's an unnecessary description.
I deleted unnecessary description.